Modules/SubModule_Import/xASL_imp_NII2BIDS_Session.m

```diff
@@ -0,0 +1,71 @@
+function xASL_imp_NII2BIDS_Session(imPar, bidsPar, studyPar, listSessions, nameSubjectSession, bidsLabel, iSession)
+%xASL_imp_NII2BIDS_SubjectSession NII2BIDS conversion for a single sessions.
+%
+% FORMAT: xASL_imp_NII2BIDS_SubjectSession(imPar, bidsPar, studyPar, listSessions, nameSubjectSession, subjectLabel, iSession)
+% 
+% INPUT:
+%   imPar                 - JSON file with structure with import parameter (STRUCT, REQUIRED)
+%   bidsPar               - Output of xASL_imp_Config (STRUCT, REQUIRED)
+%   studyPar              - JSON file with the BIDS parameters relevant for the whole study (STRUCT, REQUIRED)
+%   listSessions          - list of sessions (CELL ARRAY, REQUIRED, e.g.: {'ASL_1'})
+%   nameSubjectSession    - name of the subject (CELL ARRAY, REQUIRED)
+%   subjectLabel          - subject label (CHAR ARRAY, REQUIRED)
+%   iSession              - Session number (INTEGER, REQUIRED)
+%
+% OUTPUT:
+%  n/a
+%                         
+% -----------------------------------------------------------------------------------------------------------------------------------------------------
+% DESCRIPTION: NII2BIDS conversion for a single sessions.
+% 
+% 1. Make a subject directory
+% 2. Iterate over runs
+%
+% -----------------------------------------------------------------------------------------------------------------------------------------------------
+% EXAMPLE:     n/a
+% __________________________________
+% Copyright 2015-2021 ExploreASL
+
+    %% Unpack bidsLabel struct
+    subjectLabel = bidsLabel.subject;
+    visitLabel = bidsLabel.visit;
+
+    %% 1. Make a subject directory
+    if length(listSessions)>1 || ~isempty(visitLabel)
+        if ~isempty(visitLabel)
+            sessionLabel = ['ses-' visitLabel];
+        else
+            sessionLabel = ['ses-' listSessions{iSession}(5:end)];
```

Comment:
I don't get this fallback, why the `5:end`? Does that always work? I'm not sure if BIDS allows a single session as folder structure, I would test this in the BIDS validator, but it should be fine to always add a session folder by default.

Reply:
TBH I don't know. We can ask @jan-petr about that part when he's back from vacation. I found a small bug anyway and will debug now.

Reply:
OK, if it's indeed iterating over sessions and not SubjectSession (what is very confusing as throughout the processing modules, we refer to SubjectSession as what is SubjectRun in BIDS... :)

Reply:
Don't remember the history and we have to revamp this whole import to really use correctly sessions and not visits. 

Here, obviously, if the visitLabel is missing, it extracts the visit from the session label by removing the first four characters (=="ses-") from the session label.

Again - the sessions/visit run methodology has to be unified for BIDS - I have created new issues for that in 1.9.0. #775